cli/pkg/kctrl/cmd/package/available/get.go

```diff
@@ -65,6 +68,7 @@ func NewGetCmd(o *GetOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Command
 	}
 
 	cmd.Flags().BoolVar(&o.ValuesSchema, "values-schema", false, "Values schema of the package (optional)")
+	cmd.Flags().BoolVar(&o.GenerateDefaultValues, "generate-default-values-file", false, "Generate default values from schema of the package (optional)")
```

Comment:
we have a convention that flags that end with -output output something, so for this case, `default-values-file-output file.yml` would be better.

Reply:
Just to make sure that I have understood this correctly, we want to update the flag to `default-values-file-output` and it should accept a file name where the values should be saved, right?
I have been thinking about the latter part as well (user should have the flexibility to choose file name).